sympy/parsing/tests/test_latex.py

```diff
@@ -273,6 +273,7 @@ def test_import():
     (r"\log_2 x", _log(x, 2)),
     (r"\log_a x", _log(x, a)),
     (r"5^0 - 4^0", _Add(_Pow(5, 0), _Mul(-1, _Pow(4, 0)))),
+    (r"3 x - 1", _Add(_Mul(3, x), -1))
```

Comment:
Is this the actual srepr? I would expect something like `_Add(_Mul(3, x), Mul(-1, 1)))`.